src/test/java/org/openhab/tools/analysis/checkstyle/test/BundleVendorCheckTest.java

```diff
@@ -0,0 +1,94 @@
+package org.openhab.tools.analysis.checkstyle.test;
+
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+import com.puppycrawl.tools.checkstyle.TreeWalker;
+import com.puppycrawl.tools.checkstyle.api.Configuration;
+import org.junit.Before;
+import org.junit.Test;
+import org.openhab.tools.analysis.checkstyle.BundleVendorCheck;
+import org.openhab.tools.analysis.checkstyle.api.AbstractStaticCheckTest;
+
+/**
+ * Tests for {@link BundleVendorCheck}
+ *
+ * @author Martin van Wingerden
+ */
+public class BundleVendorCheckTest extends AbstractStaticCheckTest {
+    private static final String TEST_DIRECTORY = "correctOpenHABSpellingInManifestTest/";
+    private DefaultConfiguration checkConfig;
+
+    @Before
```

Comment:
The test which I saw in the existing code constructed it in each test so I used the same pattern but using an @Before. IMHO it's not that important if it's @Before or @BeforeClass so I can change it.